runtime/primitives/src/currency.rs

```diff
@@ -9,7 +9,9 @@ use sp_runtime::{ArithmeticError, DispatchError, RuntimeDebug};
 use serde::{Deserialize, Serialize};
 use sp_runtime::sp_std::ops::Deref;
 
-#[derive(Encode, Decode, Eq, PartialEq, Copy, Clone, RuntimeDebug, PartialOrd, Ord, TypeInfo)]
+#[derive(
+	Encode, Decode, Eq, PartialEq, Copy, Clone, RuntimeDebug, PartialOrd, Ord, TypeInfo, CompactAs,
+)]
```

Comment:
`CompacAs` is the `#[transparent]` counter-part of `parity-scale-codec` and is being used to decode assets for the `bonded-finance` benchmarks

Reply:
TIL that, thanks :)